tf_encrypted/keras/layers/flatten.py

```diff
@@ -0,0 +1,52 @@
+"""Flatten Layer implementation."""
+
+import numpy as np
+from tensorflow.python.keras.utils import conv_utils
+
+from tf_encrypted.keras.engine import Layer
+
+
+
+class Flatten(Layer):
+  """Flattens the input. Does not affect the batch size.
+  If inputs are shaped `(batch,)` without a channel dimension, then flattening
+  adds an extra channel dimension and output shapes are `(batch, 1)`.
+  Arguments:
+      data_format: A string,
+          one of `channels_last` (default) or `channels_first`.
+          The ordering of the dimensions in the inputs.
+          `channels_last` corresponds to inputs with shape
+          `(batch, ..., channels)` while `channels_first` corresponds to
+          inputs with shape `(batch, channels, ...)`.
+          If you never set it, then it will be "channels_last".
+  """
+
+  def __init__(self, data_format=None, **kwargs):
+    super(Flatten, self).__init__(**kwargs)
+    self.data_format = conv_utils.normalize_data_format(data_format)
+
+  def build(self, input_shape):
+    self.built = True
+
+  def call(self, inputs):
```

Comment:
I suggest checking how data_format is used for this layer in tf.keras [here](https://github.com/tensorflow/tensorflow/blob/1511803cb3f7f1fe88970c406abfe8f2c07e68a0/tensorflow/python/keras/layers/core.py#L571) and following suit

Reply:
Sounds good. Just added a commit. It's now supporting channels_first.